navigation/BottomTabNavigator.tsx

```diff
@@ -68,3 +76,17 @@ function ScanGunScreen() {
         </ScanGunStack.Navigator>
     );
 }
+
+const SearchStack = createStackNavigator<TabThreeParamList>();
+
+function SearchScreen() {
+    return (
+        <SearchStack.Navigator>
+            <SearchStack.Screen name="SearchScreen" component={Search} options={{ headerTitle: 'Search' }} />
+        </SearchStack.Navigator>
+    );
+}
+
+//NEW SEARCH STACK NAVIGATOR
+//SEARCH SCREENS (2X SCREEN --- SEARCH SCREEN /// SEARCH RESULT SCREEN)
+//ADD SEARCH TO BOTTOM TAB NAVIGATION
```

Comment:
Let's reserve comments for special cases that need extra context. Otherwise let's try not to commit them.